salt/utils/rsax931.py

```diff
@@ -37,6 +37,12 @@ def _find_libcrypto():
         # system.
         # look in salts pkg install location.
         lib = glob.glob("/opt/salt/lib/libcrypto.dylib")
+
+        # look in location salt is running from
+        # this accounts for running from an unpacked
+        # onedir file
+        lib = lib or glob.glob(os.path.join("lib/libcrypto.dylib"))
```

Comment:
```suggestion
        lib = lib or glob.glob("lib/libcrypto.dylib")
```
No need for `os.path.join` right?